.circleci/config.yml

```diff
@@ -73,7 +99,9 @@ workflows:
   test:
     jobs:
       - unit_test
-      - system_test
+      - system_test_e2e
```

Comment:
Which is first execution test or lint?
If test is first, I think change the order is better because lint is lighter than test.

Reply:
Test and lint run in parallel.
https://circleci.com/docs/ja/2.0/workflows/